app/models/admin_reservation.rb

```diff
@@ -10,6 +10,13 @@ class AdminReservation < Reservation
     other
   ).freeze
 
+  REPEAT_OPTIONS = %w(
```

Comment:
I feel like these probably belong on the form object if that's the only place they're used.